packages/blockchain-wallet-v4-frontend/src/components/Form/SelectBoxBitcoinAddresses/index.js

```diff
@@ -33,15 +33,17 @@ class SelectBoxBitcoinAddresses extends React.PureComponent {
 }
 
 SelectBoxBitcoinAddresses.propTypes = {
-  includeAll: PropTypes.bool
+  includeAll: PropTypes.bool,
+  includeImported: PropTypes.bool
 }
 
 SelectBoxBitcoinAddresses.defaultProps = {
-  includeAll: true
+  includeAll: true,
+  includeImported: true
 }
 
 const mapStateToProps = (state, ownProps) => ({
-  data: getData(state, ownProps.coin)
+  data: getData(state, ownProps)
```

Comment:
If you do this, also replace the argument coin with props in the getData function